config/replacements.php

```diff
@@ -4,6 +4,7 @@
     // NEVER REWRITE
     'zendframework/zendframework' => 'zendframework/zendframework',
     'Doctrine\\Zend' => 'Doctrine\\Zend',
+    'Northwoods\\Container\\Zend\\Config' => 'Northwoods\\Container\\Zend\\Config',
```

Comment:
@weierophinney 
I think we should change this rule to be "a bit more generic" rule:
```Container\\Zend => Container\\Zend```

Then it will be fine in case we have:
```
use Northwoods\Container

Container\Zend\Config::class;
```

There will be still problem if we have:
```
use Northwoods\Container\Zend;

Zend\Config::class;
```

Reply:
The early versions of the matching rules were generic, and that's when we hit issues. Having more specific rules allowed greater success.

That said, your suggestion to me in Slack about doing various rules around `a\\Zend`, `b\\Zend`, etc. makes sense, as it gives us both specificity and generality. I'll work up a patch around that later today.

Reply:
See #40